src/scripts/upload_impacted_targets.sh

```diff
@@ -2,9 +2,13 @@
 
 set -euo pipefail
 
-# API Token
-if [[ -z ${API_TOKEN+x} ]]; then
-	echo "Missing API Token"
+# API Token is required if PR is not from a fork, or
+# RUN ID is required if PR is from a fork
+if [[ (-z ${API_TOKEN-}) && (${IS_FORK} == 'false') ]]; then
```

Comment:
It looks like the action was still allowing the request through when the API token wasn't provided